lib/plugins/features/movies/controller.js

```diff
@@ -27,3 +30,19 @@ exports.list = (query) => {
   })
   .fetchAll();
 };
+
+exports.addLocation = (payload) => {
+  return Bluebird.all([
+    new Location({ id: payload.location_id }).fetch(),
+    new Movie({ id: payload.movie_id }).fetch()
```

Comment:
Because you expect both the movie id and the location id to be valid, you should add a [`require: true`](http://bookshelfjs.org/#Model-instance-fetch) to both of these `.fetch` calls. That way, it will throw a [specific error](http://bookshelfjs.org/#Model-static-NotFoundError) that you can then subsequently catch and throw a more appropriate error with [Boom](https://github.com/hapijs/boom), like a 404.